lib/v0_bindings_extensions.rb

```diff
@@ -83,11 +83,21 @@ def self.create_from_query_result(query_result)
 
   Api::V0::Bindings::GetHighlightsParameters.class_exec do
     def query(user_id:)
-      highlights = ::Highlight.by_user(user_id)
+      users = []
```

Comment:
`users` name implies this is an array of `User`.  Recommend `user_ids = []` instead.